index.html

```diff
@@ -16,7 +16,7 @@
           <nav id="nav">
           
             <a class="logo" href="#">My Logo</a> 
```

Comment:
- [ ] Good job making some changes to the logo as it was suggested by the last reviewer, but I think it still needs more work then , please fix the font family it should be as in the design .
![image](https://user-images.githubusercontent.com/74744787/219703694-b6659f81-79ab-41b6-b507-9743df87b6a6.png)

Reply:
ok, in my next commit. I am going to use the same css properties of figma. so , please consider that because I had many reviews of you but I hope you consider that I am putting the same properties of figma of microverse for this project